src/Sylius/Bundle/ApiBundle/EventHandler/ShopUserRegisteredHandler.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\EventHandler;
+
+use Sylius\Bundle\ApiBundle\Command\SendShopUserVerificationEmail;
+use Sylius\Bundle\ApiBundle\Event\ShopUserRegistered;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Model\ShopUserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Sylius\Component\User\Security\Generator\GeneratorInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/** @experimental */
+final class ShopUserRegisteredHandler
```

Comment:
```suggestion
final class ShopUserRegisteredHandler implements MessageHandlerInterface
```
We can think about dropping these interfaces, but probably all together